pom.xml

```diff
@@ -130,6 +130,7 @@
         <module>gateway-shell-release</module>
         <module>gateway-test-release-utils</module>
         <module>gateway-service-nifi</module>
+        <module>gateway-service-nifi-registry</module>
```

Comment:
I'm pretty sure we need a change in `gateway-release/pom.xml` to make sure the new module is packaged. Check for other places where the `gateway-service-nifi` module is referenced. Should be similar to that.

If you run `mvn -T.5C clean package -DskipTests -Ppackage,release` with your change, after building you should see `target/1.4.0-SNAPSHOT/knox-1.4.0-SNAPSHOT.tar.gz` and if you extract it make sure you see `data/services/nifi-registry` and `lib/gateway-service-nifi-registry*.jar`